src/main/osd/osd.c

```diff
@@ -91,6 +91,16 @@
 #include "hardware_revision.h"
 #endif
 
+#ifndef CORE_TEMPERATURE_THRESHOLD
+// a temperature above 70C should produce a warning, lockups have been reported above 80C.
+
+#ifdef STM32H7
+#define CORE_TEMPERATURE_THRESHOLD 85 // A 400Mhz+ CPU on small 27x27mm PCB generates and retains more heat than other processors.
```

Comment:
And this should include the value 70 and 80.

Reply:
ahh oh, i think understand now.

Reply:
see comments below.